salt/kube-apiserver/apiserver.jinja

```diff
@@ -50,10 +50,13 @@ KUBE_API_ARGS="--advertise-address={{ salt.caasp_net.get_primary_ip() }} \
                --tls-private-key-file={{ pillar['ssl']['kube_apiserver_key'] }} \
                --cert-dir=/etc/pki \
                {{ salt.caasp_pillar.get('components:apiserver:args') }} \
+               --requestheader-allowed-names=aggregator \
                --requestheader-username-headers=X-Remote-User \
                --requestheader-group-headers=X-Remote-Group \
                --requestheader-extra-headers-prefix=X-Remote-Extra \
                --requestheader-client-ca-file={{ pillar['ssl']['ca_file'] }} \
+               --proxy-client-cert-file={{ pillar['ssl']['kube_proxy_crt'] }} \
+               --proxy-client-key-file={{ pillar['ssl']['kube_proxy_key'] }} \
```

Comment:
Regarding the certificates, @kiall made a comment regarding this on this PR: https://github.com/kubic-project/salt/pull/666

So I don't think this is ok to reuse the kube-proxy certificate.